saleor/graphql/schema.graphql

```diff
@@ -256,6 +272,42 @@ input AppInput {
   permissions: [PermissionEnum]
 }
 
+type AppInstall {
+  errors: [Error!]! @deprecated(reason: "Use typed errors with error codes. This field will be removed after 2020-07-31.")
+  appErrors: [AppError!]!
+  appJob: AppOngoingInstallation
+}
+
+input AppInstallInput {
+  appName: String
+  manifestUrl: String
+  activateAfterInstallation: Boolean = true
+  permissions: [PermissionEnum]
+}
+
+enum AppJobStatus {
```

Comment:
@IKarbowiak is working on CSV import/export and there we have generic enum for job statuses, couldn't we use it here? I'm trying to avoid a situation when we merged these two branches and end up with the same enum doubled in the schema with different names. 

https://github.com/mirumee/saleor/blob/add-products-csv-export/saleor/graphql/schema.graphql#L2035

Reply:
Good point. `JobStatusEnum` is on master. https://github.com/mirumee/saleor/blob/master/saleor/graphql/core/enums.py#L79

Reply:
Now should be fine